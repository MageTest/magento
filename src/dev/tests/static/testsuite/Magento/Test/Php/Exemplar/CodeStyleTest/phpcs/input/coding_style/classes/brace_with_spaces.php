<?php
class Magento_Test_Php_Exemplar_CodeStyleTest_phpcs_input_coding_style_classes_brace_with_spaces
    {
}
