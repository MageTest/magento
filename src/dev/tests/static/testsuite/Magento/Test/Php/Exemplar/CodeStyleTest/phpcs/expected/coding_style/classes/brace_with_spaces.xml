<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="3"/>
</config>
